operation-domain/src/main/java/org/sopt/makers/operation/auth/domain/TeamOAuthInfo.java

```diff
@@ -0,0 +1,24 @@
+package org.sopt.makers.operation.auth.domain;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+
+import static jakarta.persistence.GenerationType.IDENTITY;
+
+@Entity
+public class TeamOAuthInfo {
+    @Id
+    @GeneratedValue(strategy = IDENTITY)
+    private Long id;
+    @Column(name = "client_id", nullable = false)
+    private String clientId;
+    @Column(name = "redirect_uri", nullable = false)
+    private String redirectUri;
+    @Column(name = "team", nullable = false)
+    @Enumerated(EnumType.STRING)
+    private Team team;
+}
```

Comment:
`P4`

가독성을 위한 개행!!
```suggestion
public class TeamOAuthInfo {

    @Id
    @GeneratedValue(strategy = IDENTITY)
    private Long id;
    
    @Column(name = "client_id", nullable = false)
    private String clientId;
    
    @Column(name = "redirect_uri", nullable = false)
    private String redirectUri;
    
    @Column(name = "team", nullable = false)
    @Enumerated(EnumType.STRING)
    private Team team;
    
}
```